### PR TITLE
Message formatting

### PR DESCRIPTION
**Pull request by or-else**
Looks much nicer now: 
Simulator Screen Shot - iPhone Xʀ - 2019-04-23 at 19 19 35

### CODE REVIEW

File: TinodeSDK/Tinode.swift [hunk 1]
```diff
@@ -285,13 +292,23 @@ public class Tinode {
...
             }
         } else if let meta = serverMsg.meta {
+            let updated: Date?
             if let t = getTopic(topicName: meta.topic!) {
                 //t.route
                 t.routeMeta(meta: meta)
+                updated = t.updated
             } else {
-                _ = maybeCreateTopic(meta: meta)
+                let t = maybeCreateTopic(meta: meta)
```
Comment (aforge) [inline TinodeSDK/Tinode.swift]: if let t = maybeCreateTopic(meta: meta) {
    self.updated = t.updated
}

File: Tinodios/MessageViewController.swift [hunk 2]
```diff
@@ -172,6 +160,8 @@ extension MessageViewController {
     func displayChatMessages(messages: [StoredMessage]) {
         self.messages = messages.reversed()
         self.messagesCollectionView.reloadData()
+        // FIXME: don't scroll to bottom in response to loading the next page.
+        // Maybe don't scroll to bottom if the view is not at the bottom already.
```
Comment (aforge) [inline Tinodios/MessageViewController.swift]: Right. We need to know the current state of the VC.
Scrolling to bottom is only needed upon the first load of the data.

File: README.md [hunk 3]
```diff
@@ -1,45 +1,36 @@
-# Tinodios
+# Tinodios: Tinode Messaging Client for iOS 
 
-Experimental iOS client.
+iOS client for [Tinode](https://github.com/tinode/chat) in Swift.
```
[the remark this answers is not in the document]
Comment (or-else) [reply README.md]: :)

File: TinodeSDK/Tinode.swift (hunk 1 above)
Comment (or-else) [reply TinodeSDK/Tinode.swift]: done

**Pull request merged**